.github/workflows/auto_publish_release.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         run: yarn install
 
       - name: Build
-        run: yarn build --optimize
+        run: yarn build
```

Comment:
Why have we dropped the `--optimize` flag?

Reply:
That was a vue-cli flag; there's no such flag for vite